tools/codegen/pkg/modelgen/model.go

```diff
@@ -217,6 +220,17 @@ func (m *Model) addTopLevelFields(fd *FileDescriptor) {
 	m.Comment = fmt.Sprintf("%s\n%s", fd.getComment(syntaxPath), fd.getComment(packagePath))
 }
 
+func validateTmplName(name string) error {
```

Comment:
Do we need other checks here? See which characters are valid in a Go symbol name. No period for example...